<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_f_obj">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Entrer la fonction objectve (ex: 3*x1 + 2*x2 ...)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="obj_function">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_constraints">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Les contraintes (ex: 2*x1 - 3*x2 &gt;= 5 ...une par ligne...) </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="constraints_data">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>381</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_constraint_limits">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>limites de contrainte (ex: x1 &gt;= 0, x2 &gt;= 0....)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="simple_constraint">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>381</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="simplex_calcul">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>710</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcul Simplexe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>710</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="max">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>maximisation</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="min">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>660</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>minimisation</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="response">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>401</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="response_label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reponses:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
